### PR TITLE
Remove links to git.sagemath.org

### PR DESCRIPTION
**Pull request by mkoeppe**
... in favor of linking to GitHub.

@dimpase

### CODE REVIEW

File: src/development-status.html [hunk 1]
```diff
@@ -4,6 +4,10 @@
 {% block content %}
  <h1 class="narrow">Status of Development</h1>
 
+ <h2 class="narrow">Release Tour for the current development series</h2>
+
+ The <a href="https://wiki.sagemath.org/ReleaseTours">release notes for the current development series</a> are developed by collaborative editing on the Sage Wiki.
```
Comment (dimpase) [inline src/development-status.html:9]: Missing closing `</a>` tag?

File: src/development-status.html [hunk 2]
```diff
@@ -16,11 +20,10 @@ <h2 class="narrow">Trac</h2>
...
       Trac is the organization tool for developing new code. Below, a quick view what&#39;s
-      currently happening. For more detailed information click
-         <a href="https://trac.sagemath.org/timeline">here</a>,
-         <a href="https://git.sagemath.org/sage.git/">here</a>
-         or read the
-         <a href="https://trac.sagemath.org/">introduction</a>.
+      currently happening. For more detailed information see
+      <ul>
+      <li> <a href="https://trac.sagemath.org/timeline">Trac development timeline</a>,
+      <li> <a href="https://trac.sagemath.org/">Trac main page</a>.
```
Comment (dimpase) [inline src/development-status.html:26]: Add `</ul>` here?

File: src/download-source.html [hunk 3]
```diff
@@ -54,9 +43,6 @@ <h2 class="narrow">Information</h2>
   <p>
     See also <a href="https://wiki.sagemath.org/ReleaseTours/sage-{{version_src}}">Release Tour for version {{version_src}}</a> and the <a href="changelogs/sage-{{version_src}}.txt">changelog</a>.<br />
   <p>
-  You can <a href=
-  "https://git.sagemath.org/">browse all the
-  tracked source code repositories</a> and see exactly what&#39;s going on, and who did what
-  when.<br />
+  You can <a href="https://github.com/sagemath/sage/">browse the Sage repository on GitHub</a>.<br />
```
Comment (dimpase) [inline src/download-source.html:46]: Say something about other Sage parts in https://github.com/sagemath/

**Review by dimpase — CHANGES_REQUESTED** [review]
could you look into these trivial changes?

File: src/development-status.html (hunk 2 above)
Comment (haraldschilly) [reply src/development-status.html:26]: yes

File: src/download-source.html (hunk 3 above)
Comment (haraldschilly) [reply src/download-source.html:46]: well, do you want to add something as part of this PR? as you wish. you can make another one later as well.

Comment (mkoeppe) [reply src/download-source.html]: done

**Comment by haraldschilly** [comment]
so, this looks good to me, I'll merge it right away and add that missing `</ul>` tag. then I'll check it again and it should be online.

**Comment by mkoeppe** [comment]
Thanks.

**Comment by haraldschilly** [comment]
ok, all done 👍🏻

**Pull request merged**